cmd/posix.go

```diff
@@ -18,6 +18,7 @@ package cmd
 
 import (
 	"context"
+	"encoding/json"
```

Comment:
We can perhaps use jsoniter instead of encoding/json? @krishnasrinivas

Reply:
done